src/DynamoCore/Core/CustomNodeManager.cs

```diff
@@ -304,7 +304,7 @@ public IEnumerable<CustomNodeInfo> AddUninitializedCustomNodesInPath(string path
         ///     Does not instantiate the nodes.
         /// </summary>
         /// <returns>False if SearchPath is not a valid directory, otherwise true</returns>
-        private IEnumerable<CustomNodeInfo> ScanNodeHeadersInDirectory(string dir, bool isTestMode)
+        private IEnumerable<CustomNodeInfo> ScanNodeHeadersInDirectory(string dir, bool isTestMode, bool isPackageMember)
```

Comment:
Please remove this `bool isPackageMember` and set the `info.IsPackageMember = isPackageMember` in the `AddUninitializedCustomNodesInPath` above.